src/pages/rama-issue/issue.css

```diff
@@ -0,0 +1,44 @@
+.image1{
+    max-width: 200px;
+    max-height: 200px;
+    float: right;
+}
+
+.text1{
+font-size: 50px;
+line-height: 75px;
+}
+
+.text2{
+    font-size: 20px;
+line-height: 30px;
+}
+
+.text5{
+    color: rgba(0, 0, 0, 0.54);
+    font-size: 18px;
+}
+
+.text6{
+    font-size: 20px;
+    line-height: 30px;
+margin-left: 60px;
```

Comment:
A margin of 60px on small screens is too much and it has no margin-right, so this will not look good and it is not that responsive.

This applies to all the margin-left in this style sheet.
A better way is to use media query to fit multiple screen sizes.

In here, it is better to use padding instead of margin

"In general, use margins when you’re adjusting the spacing of an element in relation to another element (i.e a div in relation to another div on the page), and padding when you’re adjusting the look of an individual element (i.e the amount of pixels between the edge of a div and the text within it)."

https://blog.hubspot.com/website/css-margin-vs-padding#:~:text=In%20general%2C%20use%20margins%20when,and%20the%20text%20within%20it).